tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "noImplicitAny": true,
+    "module": "commonjs",
+    "target": "es5",
+    "jsx": "react",
+    "rootDir": "src",
+    "typeRoots": ["./src/types", "node_modules/@types"]
```

Comment:
Can we install the types from `@types/ui-box` instead of including it directly in the repo?

Reply:
As mentioned above, they're not complete. The `@types/ui-box` already had some issues with transferring css types over to the created Box element and `Evergreen` uses a lot of partial imports from `ui-box` in their prop-types (which the current `@types` version doesn't support).